.envrc

```diff
@@ -88,7 +88,12 @@ export PATH
 export FISSILE_CONFIG_PREFIX=scf
 
 # We will persist compilation results to the VM here
-export SCF_PACKAGE_COMPILATION_CACHE="${SCF_PACKAGE_COMPILATION_CACHE:-${PWD}/.fissile/compilation}"
+# Caching is disabled if SCF_DISABLE_PACKAGE_COMPILATION_CACHE is set
+if [ -z ${SCF_DISABLE_PACKAGE_COMPILATION_CACHE+x} ]; then
```

Comment:
Why aren't you checking for `SCF_DISABLE_PACKAGE_COMPILATION_CACHE` directly in https://github.com/SUSE/scf/blob/develop/make/compile#L28 instead of setting `SCF_PACKAGE_COMPILATION_CACHE` to a magic value here, and then testing for the magic value later?

Reply:
I think it's fine, it's what the `.envrc` is supposed to do - calculate your env.